system-int-test/test/cmr/system_int_test/ingest/translation_test.clj

```diff
@@ -51,9 +53,13 @@
     (testing (format "Translating %s to %s" (name input-format) (name output-format))
       (let [input-str (umm-spec/generate-metadata test-context expected-conversion/example-collection-record input-format)
             expected (expected-conversion/convert expected-conversion/example-collection-record input-format output-format)
+            expected (update-in-each expected [:Platforms] update-in-each [:Instruments] assoc
```

Comment:
Because expected itself can not fix the issue.  depending on where it's converted from, expected might or might not contain the numberofinstruments. You can set it to nil there, but you still have to set nil to the actual here. I found it more clear to do them in pairs so it's clear that we are not comparing this.